gateway-service-impl/src/main/java/org/hypertrace/gateway/service/GatewayServiceImpl.java

```diff
@@ -134,6 +142,14 @@ public GatewayServiceImpl(Config appConfig) {
             entityIdColumnsConfigs);
     this.logEventsService =
         new LogEventsService(queryServiceClient, qsRequestTimeout, attributeMetadataProvider);
+    initMetrics();
+  }
+
+  private void initMetrics() {
+    serviceResponseErrorCounter =
+        PlatformMetricsRegistry.registerCounter(ERROR_COUNTER_NAME, ImmutableMap.of());
```

Comment:
If we go with the above metrics - https://github.com/hypertrace/gateway-service/pull/128#discussion_r854786951, you may have to register a counter with
```
serviceResponseErrorCounter = PlatformMetricsRegistry.registerCounter("hypertrace.gateway.requests.status", ImmutableMap.of("code", "fail"))
```

And,
```
serviceResponseSuccessCounter =
        PlatformMetricsRegistry.registerCounter("hypertrace.gateway.requests.status", ImmutableMap.of("code", "success"))
```

Reply:
Yes. Making the change.